src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx

```diff
@@ -0,0 +1,176 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import Button from 'app/components/button';
+import {PanelItem} from 'app/components/panels';
+
+const STEPS = {
+  tag: 'Tag an error',
+  repo: 'Link to a repo',
+  commit: 'Associate commits',
+  deploy: 'Tell sentry about a deploy',
+};
+
+class ReleaseProgress extends AsyncView {
```

Comment:
You should use `AsyncComponent` - the difference between `View` and `Component` is that `View` sets a document title. Generally, if your component is accessed via a route (as opposed to a component that is used to compose a "view"), then it should be a view and have a page title.